tests/test_data_cloud.py

```diff
@@ -628,8 +628,7 @@ def _test(self):
         url = get_local_url()
         self.main(["remote", "add", "-d", TEST_REMOTE, url])
 
-        self.dvc.add(self.FOO)
-        stage = self.dvc.run(deps=["foo"], outs=["bar"], cmd="echo bar > bar")
+        stage = self.dvc.run(outs=["bar"], cmd="echo bar > bar")
```

Comment:
How is this related to the patch?

Reply:
Well, its not that much, however this test was failing due to logic in Repo._collect_used_cache. I decided to simplify this test. DO you want me to revert it?

Reply:
@pared Yes, let's at least not touch that one silently within this PR :slightly_smiling_face: